Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/ReflectedPropertyEditor.cpp

```diff
@@ -515,7 +515,7 @@ namespace AzToolsFramework
 
                     // Initialized normally if the group does not have a member variable attached to it,
                     // otherwise initialize it as a group that will have a toggle switch.
-                    if (groupElementData->IsClassElement())
+                    if (groupElementData->IsClassElement() || !groupSourceNode)
```

Comment:
Can you take the opportunity to rename `groupSourceNode` to `m_groupSourceNode` to adhere to the coding guidelines

Reply:
Resolved!